frontend/app/controllers/create-badges.js

```diff
@@ -49,7 +49,7 @@ export default Controller.extend({
       let csv_ = this.get('store').createRecord('csv-file', {
         uid,
         csvFile   : csvData,
-        extension : '.csv'
+        extension : 'csv'
```

Comment:
just curious, but is this not causing upload error on backend?

Reply:
I have the same question

Reply:
I don't think so as it is working and tested also refer : https://github.com/fossasia/badgeyay/blob/fc7ac4e999d1c3b9417d7e5af4ddc19a9ace64b1/api/controllers/fileUploader.py#L66

Reply:
Oh! It has been corrected before. Sorry! I did not see that/
:+1: